virtual_rainforest/models/hydrology/above_ground.py

```diff
@@ -216,7 +216,7 @@ def calculate_drainage_map(grid: Grid, elevation: np.ndarray) -> dict[int, list[
     return dict(enumerate(upstream_ids))
 
 
-def estimate_interception(
+def calculate_interception(
```

Comment:
Is this calculating the intercept for a linear equation? Should it be `calculate_intercept` instead?

Reply:
Could you please rephrase this questions?

Reply:
By interception do you mean the point at which the line crosses the y axis? In that case I think it should be "intercept". If you mean something else, that's obvs fine!

Reply:
Ah ok, now I see. I mean 'canopy interception' which is the rainfall that is intercepted by the canopy of a tree and successively evaporates from the leaves or fall as throughfall or stemflow on the forest floor.